intl/msg_hash_us.h

```diff
@@ -6151,55 +6151,55 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_SUBLABEL_CONTENT_SHOW_FAVORITES,
-   "Show the 'Favorites' menu. (Restart Required on Ozone/XMB)"
+   "Show the 'Favorites' menu. (Restart required on Ozone/XMB)"
```

Comment:
s/Show the 'Favorites' menu. (Restart required on Ozone/XMB)/Show the 'Favorites' menu (Restart required on Ozone/XMB)./